1 intro tast/main.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  main.swift
+//  1 intro tast
+//
+//  Created by Belka on 2/2/20.
+//  Copyright © 2020 Tanya. All rights reserved.
+//
+
+import Foundation
+
+let mathController = MathController()
+let argCount = CommandLine.argc
+var introMessage = "Thank you for waiting."
+var consoleAnswer = ""
+var errorMessage = ""
+var numbers = [Double]()
+var isError = false
+
+if argCount < 2  {
+    consoleAnswer = "Please, write some numbers for counting"
+} else {
+    var arguments = CommandLine.arguments
+    for index in 1..<Int(argCount) {
+        if let newNumber = Double(arguments[index]) {
+            numbers.append(newNumber)
+        } else {
+            errorMessage = "Sorry, only digits are needed: problems with  \"\(arguments[index])\" "
```

Comment:
```suggestion
            errorMessage = "Sorry, only numbers are needed: problems with  \"\(arguments[index])\" "
```

What about numbers? :)
Your messages should be absolutely clear for the user.

Reply:
I got it. You right, thank you.